libs/internal/include/launchdarkly/data_model/context_kind.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <boost/serialization/strong_typedef.hpp>
+
+#include <string>
+
+namespace launchdarkly::data_model {
+
+BOOST_STRONG_TYPEDEF(std::string, ContextKind);
```

Comment:
It is very nice that this exists.

Reply:
I suppose I could have used this in place of my tag types for flags/segments.